src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/StringFormatConverter.cs

```diff
@@ -17,7 +17,7 @@ public class StringFormatConverter : IMultiValueConverter
         public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
         {
             string formatString = values[0] as string;
-            var formattedString = string.Format(formatString, values.Skip(1).ToArray());
+            var formattedString = string.Format(CultureInfo.CurrentCulture, formatString, values.Skip(1).ToArray());
```

Comment:
(Preferred) Either remove all calls to this converter and use [StringFormat property in all bindings](https://docs.microsoft.com/en-us/dotnet/api/system.windows.data.bindingbase.stringformat?view=netframework-4.7.2#examples) or,

https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/NuGet/NuGet%5C.Client%24+StringFormatConverter&patternType=literal

use culture from param

```suggestion
            var formattedString = string.Format(culture, formatString, values.Skip(1).ToArray());
```

@donnie-msft Do you have any reason to keep this converter?

Reply:
- I would guess there's negligible difference in terms of Perf between a converter and the direct `string.Format` you suggest. 
- Doing a quick search, I see many repos use this converter as a way to add parameter validation and/or exception handling (we're not, probably should)
- IMO, keeping this in the converter is a bit more concise than repeating StringFormat in the XAML on each binding.

Reply:
On second look, the repos I'm seeing haven't been touched in a while. @dominoFire the only reason I can think of is for validation/exception handling I mentioned.

Reply:
I'm going to keep this converter and add validation for values length.